.github/workflows/check.yml

```diff
@@ -12,6 +12,8 @@ jobs:
 
       # Use the jq tool (included in GitHub runner environments) to simplistically validate JSON files.
       - run: |
+          npm ci
           cd protocol
           jq empty errors.json
           jq empty errorsHelp.json
+          npm t
```

Comment:
I'm a firm believer that commands in scripts should always use the longest (most readable) form, so `npm test` rather than `npm t` (assuming that's what is being run here)

Reply:
agreed, this was just force of habit on my part 06a6694